### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by songofroland — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+```
+$ cd pt-dijkstra
+$ yarn start
+```
+###### NOTE: You need to have `yarn` installed.
+
+This should open browser. At the moment we don't have live demo. The site is mobile friendly so it should work on any screen size.
+
+## Graphs
+Simulator provides few graphs you can try.
```
[the remark this answers is not in the document]
Comment (wilmhit) [reply README.md:18]: no. a few and few are different things

File: README.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+The browser does the heavy lifting, so performance will highly depend on device used.
+
+###### - Is there a way to save graphs?
+###### - Of course! Right click on input field > Select all > Copy
+
+
+## Used technologies
+ - Typescript
+ - React
+ - yarn
```
[the remark this answers is not in the document]
Comment (wilmhit) [reply README.md:32]: You won't run it without it

**Comment by wilmhit** [comment]
Nice job with grammar/typos check

File: README.md [hunk 3]
```diff
@@ -1 +1,56 @@
...
+
+Since we use Typescript compailer you won't be able to understand generated js looking at it in browser.
+
+To debug react there is extension available called "React developer tools" for most popular browsers.
+
+ESLint was used to control coding style. It will generate warnings and errors in terminal after `yarn start` command if code is not styled correctly.
+
+## Hacking
+Want to see how it is build? Or maybe you have different algorithm to show? Either way the code is here for you to modify.
```
[the remark this answers is not in the document]
Comment (wilmhit) [reply README.md:41]: I don't think there it is possible(without much effort) to show more than one algotithm at a time

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (songofroland) [reply README.md:19]: yeah I know, I'd still use a few for 3 graphs (looks better)

**Pull request merged**